### DISCUSSION TITLE
Uploading images in Github project issues as comments - will this take up the storage limit of repos/project?

### DISCUSSION THREAD

**Discussion by jialinlow-bein** [community/community#69970] [Projects and Issues]
### Select Topic Area

General

### Feature Area

Issues

### Body

If i were to upload pictures in Github project issues as comments, does it take up the 2GB storage limit of the free plan? 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by santoshvandari** [comment]
Yes, uploading images in GitHub project issues as comments will count towards the storage limit of the repository, including the 2GB storage limit for the free plan.

GitHub's storage limit includes all the files and data associated with the repository, such as source code, issues, pull requests, images, and other files uploaded. This limit is a total cap on the storage used by the entire repository.

So, if you upload images as comments in issues, these images will consume storage space within the repository and count towards the 2GB storage limit on the free plan. If you exceed the storage limit, you may need to consider reducing the size of the repository or upgrading to a paid plan with higher storage limits.

It's essential to be mindful of the storage limit when using GitHub repositories, especially in free plans, and consider optimizing storage resources to ensure you don't run into storage-related issues.

**Reply by Eiim** [reply]
Can you provide a source for this? No offense but it just sounds like a ChatGPT-generated response.

**Reply by mohit-nagaraj** [reply]
idk abt resticstions on image, but one of my friends was working on backend and tried upload a ai model, but then storage limit was reached telling some gb. I am not sure about this for images. But the advantage is no. of repo's aren't limited.

**Reply by santoshvandari** [reply]
Yeah, It is. To Follow the Standard writing procedure. If you think I have done wrong, Then apologize for this. 
There is One More Way to Upload Images in the Comments just by hosting those images on the other side and linking those images in the Comments.

**Comment by Eiim** [comment]
I don't think there is such a 2GB limit for repositories. You might be thinking of the [2GB LFS object limit](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-git-large-file-storage), but that's quite different. There's apparently a [soft limit](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github#repository-size-limits) of 5GB/repo, but unless you have large binaries in your repository, you're unlikely to get anywhere close to that. Since there is no hard cap, and what limits there are are very high, it's unlikely that you'll need to worry about attachments to issues. That said, I would be quite surprised if they counted against you, as anyone can create them, and they're uploaded to GitHub's CDN rather than being part of your project folder and tracked in `.git`.
